libs/community/langchain_community/document_loaders/parsers/pdf.py

```diff
@@ -235,14 +235,20 @@ def get_image(layout_object: Any) -> Any:
 
         images = []
 
-        for img in list(filter(bool, map(get_image, page))):
-            if img.stream["Filter"].name in _PDF_FILTER_WITHOUT_LOSS:
+        for img in filter(bool, map(get_image, page)):
+            img_filter = img.stream["Filter"]
+            if isinstance(img_filter, list):
+                filter_names = [f.name for f in img_filter]
+            else:
+                filter_names = [img_filter.name]
+
+            if any(name in _PDF_FILTER_WITHOUT_LOSS for name in filter_names):
```

Comment:
if both are passed, should we throw an error that only one or the other should be passed?